src/index.js

```diff
@@ -1,3 +1,18 @@
+// handles keypresses for accessibility
+
+function handleClick(e) {
+	if (e.type === 'keypress') {
+		const code = e.keyCode;
+		if (code === 32 || code === 13) {
```

Comment:
One last thing I just noticed, I think we could swap this if statement around and just use a return. This would return true when we need it to, and false otherwise. How's this look to you?
```suggestion
		return code === 32 || code === 13;
```

Reply:
That looks good to me. 👍